dialogue-core/src/main/java/com/palantir/dialogue/core/FixedLimitedChannel.java

```diff
@@ -35,12 +36,14 @@
 
     private final LimitedChannel delegate;
     private final AtomicInteger usedPermits = new AtomicInteger(0);
+    private final Meter limitedMeter;
     private final int totalPermits;
     private final Runnable returnPermit;
 
-    FixedLimitedChannel(LimitedChannel delegate, int totalPermits) {
+    FixedLimitedChannel(LimitedChannel delegate, int totalPermits, DialogueClientMetrics metrics) {
         this.delegate = delegate;
         this.totalPermits = totalPermits;
+        this.limitedMeter = metrics.limited("FixedLimitedChannel");
```

Comment:
`FixedLimitedChannel.class.getSimpleName()` ? Then it will get auto-refactored nicely for us?